cmd/util/ledger/migrations/account_creation_migration.go

```diff
@@ -0,0 +1,243 @@
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/onflow/cadence/runtime/interpreter"
+
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// AccountCreationMigration moves account restriction logic from fvm code to the service account
+func AccountCreationMigration(payloads []ledger.Payload) ([]ledger.Payload, error) {
+	l := NewView(payloads)
+	st := state.NewState(l)
+	sth := state.NewStateHolder(st)
+	a := state.NewAccounts(sth)
+
+	address := "e467b9dd11fa00df"
+	err := migrateContractForAccount(a, address)
+	if err != nil {
+		return nil, err
+	}
+
+	key := "/storage/isAccountCreationRestricted"
```

Comment:
Is this right key, consider we call the encode function later?

Reply:
It corresponds to this: https://github.com/onflow/flow-core-contracts/blob/ab9fccdc7204968fc943db418e145a5a9638faeb/contracts/FlowServiceAccount.cdc#L111

but not sure the string formatting is ok.